oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/process/ProcessTraffic.java

```diff
@@ -0,0 +1,226 @@
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.core.analysis.manual.process;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.skywalking.oap.server.core.Const;
+import org.apache.skywalking.oap.server.core.analysis.IDManager;
+import org.apache.skywalking.oap.server.core.analysis.Layer;
+import org.apache.skywalking.oap.server.core.analysis.MetricsExtension;
+import org.apache.skywalking.oap.server.core.analysis.Stream;
+import org.apache.skywalking.oap.server.core.analysis.metrics.Metrics;
+import org.apache.skywalking.oap.server.core.analysis.worker.MetricsStreamProcessor;
+import org.apache.skywalking.oap.server.core.remote.grpc.proto.RemoteData;
+import org.apache.skywalking.oap.server.core.storage.StorageHashMapBuilder;
+import org.apache.skywalking.oap.server.core.storage.annotation.Column;
+import org.apache.skywalking.oap.server.library.util.StringUtil;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.apache.skywalking.oap.server.core.source.DefaultScopeDefine.PROCESS;
+
+@Stream(name = ProcessTraffic.INDEX_NAME, scopeId = PROCESS,
+    builder = ProcessTraffic.Builder.class, processor = MetricsStreamProcessor.class)
+@MetricsExtension(supportDownSampling = false, supportUpdate = true)
+@EqualsAndHashCode(of = {
+        "instanceId",
+        "name",
+})
+public class ProcessTraffic extends Metrics {
+    public static final String INDEX_NAME = "process_traffic";
+    public static final String SERVICE_ID = "service_id";
+    public static final String INSTANCE_ID = "instance_id";
+    public static final String NAME = "name";
+    public static final String LAYER = "layer";
+    public static final String AGENT_ID = "agent_id";
+    public static final String PROPERTIES = "properties";
+    public static final String LAST_PING_TIME_BUCKET = "last_ping";
+    public static final String DETECT_TYPE = "detect_type";
+
+    private static final Gson GSON = new Gson();
+
+    @Setter
+    @Getter
+    @Column(columnName = SERVICE_ID)
+    private String serviceId;
+
+    @Setter
+    @Getter
+    @Column(columnName = INSTANCE_ID)
```

Comment:
I think even in default, the length of instance ID is longer than 200. It is `base64(service name) + base64(instance name)`

Reply:
Refer to the [default name length in OAP configiguration](https://github.com/apache/skywalking/blob/ff1103ec1d2d7c6a1a896730cfe3024e1e99782f/oap-server/server-starter/src/main/resources/application.yml#L115-L117), It should be less than `200`. 
Maybe I could update the length to the `600`, After I calculate It could handle 200 lengths in service and instance name? I think it's enough. 
Also, I see some other metrics in OAP, they have `instanceId` field and they are using the default length(200), such as `LogRecord`, `NetworkAddressAlias` etc. Should we update too?

Reply:
I think we should update all to 600. This was missed for a long time. But, it is better to submit another separate pull request to fix those.

Reply:
update the `instanceId` in `ProcessTraffic`. I will create another PR to fix other traffic after this PR merge.